snc.sh

```diff
@@ -169,6 +169,31 @@ export OPENSHIFT_INSTALL_INVOKER="codeReadyContainers"
 
 ${OPENSHIFT_INSTALL} --dir $INSTALL_DIR create cluster --log-level debug
 
+# export the kubeconfig
+export KUBECONFIG=$INSTALL_DIR/auth/kubeconfig
+
+# Add kubelet-bootstrap-cred-manager daemonset
+oc apply -f kubelet-bootstrap-cred-manager-ds.yaml
+
+# Delete the current csr signer to get new request. 
+oc delete secrets/csr-signer-signer secrets/csr-signer -n openshift-kube-controller-manager-operator
+
+# Wait for 5 min to make sure cluster is stable again.
+sleep 300
+
+# Remove the 24 hours certs and bootstrap kubeconfig
+# This kubeconfig again get generated and new certs will be created in pki folder
+# which will have 30 days validity.
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo rm -fr /var/lib/kubelet/pki
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo rm -fr /var/lib/kubelet/kubeconfig
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl restart kubelet
+
+# Wait until bootstrap csr request is generated.
+until oc get csr | grep Pending; do echo 'Waiting for first CSR request.'; sleep 2; done
+for CSR in $(oc get csr | grep Pending | awk '{print $1}'); do
+       oc adm certificate approve $CSR;
+done
```

Comment:
The instructions you link to have an alternative to this command which involves less parsing:
`oc get csr -oname | xargs oc adm certificate approve`